webui/react/src/App.tsx

```diff
@@ -25,14 +24,17 @@ import { paths, serverAddress } from 'routes/utils';
 import Spinner from 'shared/components/Spinner/Spinner';
 import usePolling from 'shared/hooks/usePolling';
 import { StoreContext } from 'stores';
+import { useAuth, useFetchUsers } from 'stores/users';
 import { correctViewportHeight, refreshPage } from 'utils/browser';
+import { Loadable } from 'utils/loadable';
 
 import css from './App.module.scss';
 
 const AppView: React.FC = () => {
   const resize = useResize();
   const storeDispatch = useStoreDispatch();
-  const { auth, info, ui } = useStore();
+  const { info, ui } = useStore();
+  const auth = Loadable.getOrElse({ checked: false, isAuthenticated: false }, useAuth().auth);
```

Comment:
Instead of propagating this fake data, it would be better to check the actual ground truth. Normally I would recommend carrying the loadable out to the use-cases so they can each handle the loading case in the appropriate manner but here you can just make this as simple as
```typescript
const isAuthenticated = Loadable.match(useAuth(), {
  Loaded: (auth) => auth.isAuthenticated,
  NotLoaded: () => false
});
```

Reply:
There are some other places you're still using this "initial state" pattern that I recommend fixing too. `checked` should be replaceable by `Loadable.isLoaded(useAuth())`.